components/library/libfuse/patches/mount_util.c.patch

```diff
@@ -28,13 +34,15 @@
          setuid(geteuid());
 -        execl("/bin/umount", "/bin/umount", "-i", mnt, lazy ? "-l" : NULL,
 -              NULL);
+-        fprintf(stderr, "%s: failed to execute /bin/umount: %s\n", progname,
 +        if (lazy) {
-+            execle("/bin/umount", "/bin/umount", "-i", mnt, "-l",
++            execle("/sbin/umount", "/sbin/umount", "-a", mnt, 
```

Comment:
If the action is the same in both cases, why do we care about **lazy**?

Reply:
I was too lazy to clean up the lazy condition.